shared/params/minimal_config.go

```diff
@@ -76,5 +76,7 @@ func MinimalSpecConfig() *BeaconChainConfig {
 
 	minimalConfig.DepositContractTreeDepth = 32
 	minimalConfig.FarFutureEpoch = 1<<64 - 1
+	minimalConfig.HeadOfVotingPeriodLength = 6
```

Comment:
`SLOTS_PER_ETH1_VOTING_PERIOD = 32` in the minimal config, so I chose the number which is closes to `sqrt(32)`.